src/ObjectUtils/PdbFileLlvm.cpp

```diff
@@ -36,20 +47,72 @@ class SymbolInfoVisitor : public llvm::codeview::SymbolVisitorCallbacks {
     SymbolInfo symbol_info;
     symbol_info.set_name(proc.Name.str());
     symbol_info.set_demangled_name(llvm::demangle(proc.Name.str()));
+
+    // The ProcSym's name does not contain an argument list. However, this information is required
+    // when dealing with overloads and it is available in the type info stream. See:
+    // https://llvm.org/docs/PDB/TpiStream.html
+    llvm::StringRef argument_list = RetrieveArgumentList(proc);
+    if (!argument_list.empty()) {
+      symbol_info.set_demangled_name(
+          absl::StrCat(symbol_info.demangled_name(), argument_list.data()));
+    }
+
     // The address in PDB files is a relative virtual address (RVA), to make the address compatible
     // with how we do the computation, we need to add both the load bias (ImageBase for COFF) and
     // the offset of the executable section.
     symbol_info.set_address(proc.CodeOffset + object_file_info_.load_bias +
                             object_file_info_.executable_segment_offset);
     symbol_info.set_size(proc.CodeSize);
+
```

Comment:
Intended whitespace change?

Reply:
Yes, now that the filling of the `symbol_info` is more involved, I felt a whitespace here helps the reader.